USER_MANUAL.md

```diff
@@ -90,9 +83,9 @@ your radio for transmission over the air.
 Tab | Sound Device | Notes
 --- | --- | ---
 Receive Tab | Input To Computer From Radio | The off air FreeDV signal **from** your radio rig interface to your computer
-Receive Tab | Output From Computer To Speaker/Headphones | The decoded audio from your computer to your Speaker/headphones
+Receive Tab | Output From Computer To Speaker | The decoded audio from your computer to your Speaker
```

Comment:
@drowe67, does the FreeDV app itself need to be updated to remove "Headphones" from the Audio Config dialog?

Reply:
Hmmm.  This was actually an annoying hack I had to do.  When the PDF renders from markdown it can't break up table rows over multiple lines, it just truncates them.  Actually I think `Speaker/headphones` is best, we should stick with that.  I just need to reword to fit on one line or learn how to drive pandoc better :thinking:

Reply:
OK I have figured out how to do multi-line tables